src/Symfony/Component/Form/Extension/Validator/Constraints/FormValidator.php

```diff
@@ -97,6 +99,10 @@ public function validate($form, Constraint $formConstraint)
                         }
                     }
                 }
+
+                foreach ($groupedConstraints as $group => $constraint) {
```

Comment:
```diff
- constraint
+ constraints
```

?

Reply:
That would override the `$constraints` variable used above.

Reply:
Maybe name it `$constrainsInGroup` or so? Slightly confusing otherwise 🙈